server/src/main/java/org/candlepin/pinsetter/core/PinsetterKernel.java

```diff
@@ -237,35 +240,44 @@ else if (!isClustered()) {
                         }
                     }
                 }
-                // get the default schedule from the job class in case one
-                // is not found in the configuration.
-                String defvalue = PropertyUtil.getStaticPropertyAsString(jobImpl,
-                    "DEFAULT_SCHEDULE");
-
-                String schedule = this.config.getString("pinsetter." +
-                    jobImpl + ".schedule", defvalue);
-
-                if (schedule != null && schedule.length() > 0) {
-                    if (log.isDebugEnabled()) {
-                        log.debug("Scheduler entry for " + jobImpl + ": " +
-                            schedule);
-                    }
-
-                    addUniqueJob(pendingJobs, jobImpl,
-                        existingCronTriggers, schedule);
-                }
-                else {
-                    log.warn("No schedule found for " + jobImpl + ". Skipping...");
+                String schedule = getSchedule(jobImpl);
+                if (schedule != null) {
+                    addUniqueJob(pendingJobs, jobImpl, existingCronTriggers, schedule);
                 }
             }
         }
         catch (SchedulerException e) {
             throw new RuntimeException(e.getLocalizedMessage(), e);
         }
-        catch (Exception e) {
+        scheduleJobs(pendingJobs);
+    }
+
+    /* get the default schedule from the job class in case one is not found in the configuration.
+     */
+    private String getSchedule(String jobImpl) {
+        String defvalue = null;
+        try {
+            defvalue = PropertyUtil.getStaticPropertyAsString(jobImpl, "DEFAULT_SCHEDULE");
+        }
+        catch (NoSuchFieldException e) {
             throw new RuntimeException(e.getLocalizedMessage(), e);
         }
-        scheduleJobs(pendingJobs);
+        catch (ClassNotFoundException e) {
+            throw new RuntimeException(e.getLocalizedMessage(), e);
+        }
+
+        String schedule = this.config.getString("pinsetter." + jobImpl + ".schedule", defvalue);
+
+        if (schedule != null && schedule.length() > 0) {
+            if (log.isDebugEnabled()) {
+                log.debug("Scheduler entry for " + jobImpl + ": " + schedule);
+                return schedule;
```

Comment:
This only returns the schedule if debug output is enabled. We don't even really need the debug enabled check here.

Also, can we update the log statements throughout to use the substitution syntax?